plugins/ps/post-deploy

```diff
@@ -4,5 +4,8 @@ source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
 source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
+DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
+
+[[ -f "$DOKKU_PROCFILE" ]] && rm -f "$DOKKU_PROCFILE"
```

Comment:
And actually `extract_procfile` is probably a better function name than `get_procfile`. I can change it to be more explicit, and add a `cleanup_procfile` or `remove_procfile` function to make `post-deploy` cleaner.